src/export/InstanceExporter.ts

```diff
@@ -155,6 +142,98 @@ export class InstanceExporter {
     }
   }
 
+  /**
+   * Check that all required elements are present on an InstanceDefinition or
+   * a sub-part of an InstanceDefinition for all children of an element.
+   * An element is required if it has minimum cardinality greater than 1.
+   * @param {{[key: string]: any}} instance - The InstanceDefinition or subsection of an InstanceDefinition we are validating
+   * @param {ElementDefinition} element - The element we are trying to validate all children of
+   * @param {Instance} fshDefinition - The FSH definition that we built the original InstanceDefinition from
+   */
+  private validateRequiredChildElements(
+    instance: { [key: string]: any },
+    element: ElementDefinition,
+    fshDefinition: Instance
+  ): void {
+    // Get only direct children of the element
+    const children = element.children(true);
+    children.forEach(child => {
+      // Get the last part of the path, A.B.C => C
+      const childPathEnd = child.path.split('.').slice(-1)[0];
+      let instanceChild = instance[childPathEnd];
+      // If the element is a choice, we will fail to find it, we need to use the choice name
+      if (!instanceChild && childPathEnd.endsWith('[x]')) {
+        const choiceSlices = children.filter(c => c.path === child.path && c.sliceName);
+        for (const choiceSlice of choiceSlices) {
+          instanceChild = instance[choiceSlice.sliceName];
+          if (instanceChild) {
```

Comment:
What if the `instanceChild's` value is a boolean (e.g., `valueBoolean: false`)?  I think you should actually just be checking if it's defined, right?